docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  n8n:
```

Comment:
Locally, I'd also love to run n8n in docker, so that I don't need to tear down my n8n home instance. Of course I can do that differently, but this means that you don't forget when running your tests.
Do you think we should remove it from here?

Reply:
Plus: this makes it easy to create your test-workflows inside the docker; makes it easy to connect over the local docker-network, etc.